### ISSUE TITLE
System Error when running Ganache 2.0.0 on win32

### ISSUE THREAD

**Issue by itsermin** [trufflesuite/ganache#1279]
PLATFORM: win32
GANACHE VERSION: 2.0.0

EXCEPTION:
```
Error: watch D:\12 coding\02 solidity\truffle\incPurchasing\UI7\truffle.js ENOENT
    at exports._errnoException (util.js:1024:11)
    at FSWatcher.start (fs.js:1394:19)
    at Object.fs.watch (fs.js:1420:11)
    at ProjectFsWatcher.start (C:\Program Files\WindowsApps\Ganache_2.0.0.0_x64__zh355ej5cj694\app\resources\app.asar\src\truffle-integration\projectFsWatcher.js:40:29)
    at new ProjectFsWatcher (C:\Program Files\WindowsApps\Ganache_2.0.0.0_x64__zh355ej5cj694\app\resources\app.asar\src\truffle-integration\projectFsWatcher.js:25:10)
    at ProjectsWatcher.add (C:\Program Files\WindowsApps\Ganache_2.0.0.0_x64__zh355ej5cj694\app\resources\app.asar\src\truffle-integration\projectsWatcher.js:130:23)
    at process.<anonymous> (C:\Program Files\WindowsApps\Ganache_2.0.0.0_x64__zh355ej5cj694\app\resources\app.asar\src\truffle-integration\index.js:59:34)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```

**Comment by davidmurdoch** [comment]
Hey there @itsermin . This has been fixed in 2.0.1. Can you upgrade and let me know if you are still having issues?

**Comment by itsermin** [comment]
Yup it's working now, thanks!
